src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -114,6 +125,14 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
     }
   }
 
+  private updateRadioButtonTabindexes(): void {
+    if (this.radios) {
+      this.radios.forEach(radio => {
+        radio.tabindex = this.tabindex;
```

Comment:
Won't this also trigger the console.warn?

Reply:
Oh! Yes it will! Very nice catch. I'll approach this differently

Reply:
@Blackbaud-AlexKingman Should be fixed now